llvm/lib/Target/NVPTX/NVPTXISelLowering.cpp

```diff
@@ -5686,8 +5686,7 @@ PerformBUILD_VECTORCombine(SDNode *N, TargetLowering::DAGCombinerInfo &DCI) {
   return DAG.getNode(ISD::BITCAST, DL, VT, PRMT);
 }
 
-static SDValue combineADDRSPACECAST(SDNode *N,
-                                    TargetLowering::DAGCombinerInfo &DCI) {
+static SDValue PerformADDRSPACECASTCombine(SDNode *N) {
```

Comment:
I think the previous name is shorter and more consistent with LLVM stylistic standards.